tests/swift/Wasm.tests/Tests/FlatBuffers.Test.Swift.WasmTests/monster_test_generated.swift

```diff
@@ -1453,7 +1485,15 @@ public struct MyGame_Example_Monster: FlatBufferObject, Verifiable, ObjectAPIPac
     scalarKeySortedTablesVectorOffset scalarKeySortedTables: Offset = Offset(),
     nativeInline: MyGame_Example_Test? = nil,
     longEnumNonEnumDefault: MyGame_Example_LongEnum = .longone,
-    longEnumNormalDefault: MyGame_Example_LongEnum = .longone
+    longEnumNormalDefault: MyGame_Example_LongEnum = .longone,
+    nanDefault: Float32 = .nan,
+    infDefault: Float32 = +.infinity,
+    positiveInfDefault: Float32 = +.infinity,
+    infinityDefault: Float32 = +.infinity,
+    positiveInfinityDefault: Float32 = +.infinity,
```

Comment:
I believe we don't need the `+.` since we can just pass in a `.infinity` and it will be positive